core/src/main/java/jenkins/model/Jenkins.java

```diff
@@ -3986,7 +3986,7 @@ public synchronized HttpResponse doReload() throws IOException {
         LOGGER.log(Level.WARNING, "Reloading Jenkins as requested by {0}", getAuthentication().getName());
 
         // engage "loading ..." UI and then run the actual task in a separate thread
-        servletContext.setAttribute("app", new HudsonIsLoading());
+        WebApp.get(servletContext).setApp(new HudsonIsLoading());
```

Comment:
Not actually necessary but noticed while I was here that this code was failing to use the official accessor API.